widgetssdk/src/main/java/com/glia/widgets/chat/domain/GliaSendMessageUseCase.java

```diff
@@ -84,8 +87,8 @@ public void execute(SingleChoiceAttachment singleChoiceAttachment, Listener list
         chatRepository.sendMessageSingleChoice(singleChoiceAttachment, listener);
     }
 
-    public void execute(String message, SingleChoiceAttachment singleChoiceAttachment, Listener listener) {
-        chatRepository.sendMessageWithAttachment(message, singleChoiceAttachment, listener);
+    public void execute(String cardMessageId, String text, String value, Listener listener) {
```

Comment:
To my mind, it might be difficult to differentiate the purpose of these two `Strings`: `text` and `value`. Will it be a good idea to name them `message` and `answer` or `selected option` for example?

Reply:
Actually, these two parameters are the selected option. Them describes there:
https://github.com/salemove/android-sdk-widgets/pull/345/files#diff-bfe4d381d86b2e51153f365479940f0487061a1f813cf475e49f5865a3f011a8R59-R68

Reply:
Also, them has typical names for the Option values:
https://docs.glia.com/glia-dev/docs/suggestion#suggestion-with-custom-response-card